desktop/src/main/java/bisq/desktop/components/paymentmethods/MoneyBeamForm.java

```diff
@@ -75,7 +75,7 @@ public void addFormForAddAccount() {
 
     @Override
     protected void autoFillNameTextField() {
-        setAccountNameWithString(accountIdInputTextField.getText());
+        setAccountNameWithString(accountIdInputTextField.getText().trim());
```

Comment:
same as above. Should we add the "trim" to the base class `PaymentMethodForm`? or is there a single payment method that needs leading/trailing whitespaces? @ripcurlx what is your opinion?

Reply:
Here it makes sense to move the trim into the PaymentMethodForm as there is no value to pollute your account name (displayed to creator only) with additional spaces. So it would help to remove unnecessary whitespaces that were added by mistake.